packages/did/src/Did.utils.ts

```diff
@@ -158,7 +161,11 @@ const multicodecReversePrefixes: Record<DidVerificationMethodType, number> = {
 export function multibaseKeyToDidKey(
   publicKeyMultibase: VerificationMethod['publicKeyMultibase']
 ): DecodedVerificationMethod {
-  const decodedMulticodecPublicKey = multibaseDecode(publicKeyMultibase)
+  const decodedMulticodecPublicKey = base58Decode(
+    publicKeyMultibase.startsWith('z')
+      ? publicKeyMultibase.slice(1)
+      : publicKeyMultibase
+  )
```

Comment:
I think we should just throw if it does not start with z, because then we're faced with an unknown multibase encoding, or something else entirely